pom.xml

```diff
@@ -112,12 +112,14 @@
 		<benchmarkjar.name>benchmarks</benchmarkjar.name>
 		<javac.target>1.8</javac.target>
 		<slf4j.version>1.7.21</slf4j.version>
-		<log4j2.version>2.6.2</log4j2.version>
+		<log4j2.version>2.1</log4j2.version>
```

Comment:
Or u could try excluding the logging from Storm when adding Storm as a dependency. for eg: u can see how Storm is being excluded wit ElasticSearch dependency in present pom.